week-1/errors/3.js

```diff
@@ -1,6 +1,11 @@
-const cardNumber = 4533787178994213;
+const cardNumber = "4533787178994213";
 const last4Digits = cardNumber.slice(-4);
 
+console.log("Last 4 digits : ${last4Digits}");
+
+/*1.The slice method is typically used with strings to extract a portion of the string. 
+to extract the last 4 digits of the card number, we must first convert it to a string and then use the slice method.So we put the number in string */
```

Comment:
Your analysis of the problem is correct. However, the tasks asks to "_try updating the expression last4Digits is assigned to, in order to get the correct value_"

How might you update the expression to make the value that's being sliced into a string?